wp-content/themes/missilethreat/template-parts/entry-header.php

```diff
@@ -7,28 +7,137 @@
  * @since 1.0.0
  */
 
-$entry_header_classes = '';
 
+$page_for_posts = get_option( 'page_for_posts' );
 ?>
 
-<header class="single__header<?php echo esc_attr( $entry_header_classes ); ?>">
+<?php 
+if( is_single() && 'defsys' === get_post_type() || is_single() && 'missile' === get_post_type() ) { 
```

Comment:
WP actually has a built in function to help simplify this a bit. It checks if it's a single page given either a single post type or an array of post types.
```
is_singular( array( 'defsys', 'missile' ) );
```